tools/slurm/scripts/slurmcluster.sh

```diff
@@ -214,16 +204,6 @@ if [[ $OPT_CONTAINER_RUN_TYPE == "enroot" ]]; then
     fi
 fi
 
-TEMPYAML=$TEMPDIR/slurmcluster.yaml
-envsubst <$PARENT_PATH/slurmcluster.yaml >$TEMPYAML
-if [[ -n $DETERMINED_AGENT ]]; then
-    # When deploying with the determined agent, remove the resource_manager section
-    # that would otherwise be used.   This then defaults to the agent rm and
-    # the master waits for agents to connect and provide resources.
-    sed -i -e '/resource_manager/,/resource_manager_end/d' $TEMPYAML
-fi
-echo "Generated devcluster file: $TEMPYAML"
```

Comment:
https://app.circleci.com/pipelines/github/determined-ai/determined/58038/workflows/b30764bf-8db9-4b01-acba-ad1d92c0b0e5/jobs/2757463?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link&utm_content=summary

is this still getting generated? the failure might be relate to this